static/app/views/settings/projectTags.spec.jsx

```diff
@@ -38,42 +38,40 @@ describe('ProjectTags', function () {
       body: [],
     });
 
-    const wrapper = mountWithTheme(
-      <ProjectTags params={{orgId: org.slug, projectId: project.slug}} />
-    );
-
-    expect(wrapper.find('EmptyMessage')).toHaveLength(1);
+    render(<ProjectTags params={{orgId: org.slug, projectId: project.slug}} />);
+    expect(screen.getByTestId('empty-message')).toBeInTheDocument();
```

Comment:
nit: isn't it better if we check for the empty message text instead of a `test-id`?

Reply:
I'm kind of in different on these empty message text's that don't have a lot of semantic meaning (The text is kinda cute, so it feels like it would be likely to change)